MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -266,6 +267,13 @@ public void SetAvailableDecoders(IEnumerable<string> list)
             //_logger.Info("Supported decoders: {0}", string.Join(",", list.ToArray()));
         }
 
+        private List<string> _hwaccels = new List<string>();
+        public void SetAvailableHwaccels(IEnumerable<string> list)
+        {
+            _hwaccels = list.ToList();
+            //_logger.Info("Supported hwaccels: {0}", string.Join(",", list.ToArray()));
```

Comment:
```suggestion
            _logger.Debug("Supported hwaccels: {0}", string.Join(",", list.ToArray()));
```

Reply:
It is already logged in `EncodingValidator`.

https://github.com/jellyfin/jellyfin/blob/a388d5dd72893fa0a4e30472bd1d5fc8854ad51d/MediaBrowser.MediaEncoding/Encoder/EncoderValidator.cs#L259